### ISSUE TITLE
beforeitsnews.com: breakage

### ISSUE THREAD

**Issue by FewThereBe** [uBlockOrigin/uAssets#23156]
### Prerequisites

- [X] This is **NOT** a [_YouTube_](https://www.github.com/uBlockOrigin/uAssets/issues/20586), [_Facebook_](https://www.github.com/uBlockOrigin/uAssets/issues/3367), [_Twitch_](https://www.github.com/uBlockOrigin/uAssets/issues/5184) or [_a shortener/hosting site_](https://github.com/uBlockOrigin/uAssets/discussions/17361) report. _These sites MUST be reported by clicking their **respective links**_.
- [X] I read and understand the [policy about what is a valid filter issue](https://github.com/uBlockOrigin/uAssets/blob/master/README.md#uassets).
- [X] I verified that this issue is not a duplicate. (Use this [button](https://user-images.githubusercontent.com/585534/146582579-c32707a0-36a1-4cc5-ad50-83172c9f67a8.png) to find out.)
- [X] I did not remove any of the [default filter lists](https://github.com/uBlockOrigin/uAssets/assets/66517106/6e8d024a-5869-47cf-b142-b2357eb6c7d0), or I have verified that the issue was not caused by removing any of the default lists.
- [X] I did not enable additional filter lists, or I have verified that the issue still occurs without enabling additional filter lists.
- [X] I do not have custom filters/rules, or I have verified that the issue still occurs without custom filters/rules.
- [X] I am not using uBlock Origin along with other content blockers.
- [X] I have verified that the web browser's built-in content blocker/tracking protection, [network wide/DNS blocking](https://en.wikipedia.org/wiki/DNS_blocking), or my VPN is not causing the issue.
- [X] I have verified that other extensions are not causing the issue.
- [X] If this is about a breakage or detection, I have verified that it is caused by uBlock Origin and isn't a site issue.
- [ ] I did not answer truthfully to **ALL** the above checkboxes.

### URL address of the web page

`https://beforeitsnews.com/prophecy/2024/04/solar-eclipses-great-signs-in-the-heaven-x-marks-the-spot-twice-in-ten-years-2551084.html`

### Category

breakage

### Description

With uBO enabled, the "Now" counter number that is supposed to appear for each BeforeItsNews story is blanked out/it does not show up. 

The "Now" counter is supposed to be a number, giving how many people are currently viewing the particular story.

There are three other numbers/counters in the "counter box" for each story that _do_ appear correctly with uBO enabled.

It is just the one labeled "Now" that does not show up. It is blank.

And it happens on all of the BeforeItsNews stories that I have tried.

When I disable uBO, the "Now" number count re-appears correctly.

I just moved from Brave browser to Firefox, and added uBO for my ad blocker, so this is a new problem for me.

Thank you for any help. 

### Other extensions used

I currently use three others in addition to uBO: Brave Search, LastPass, and Weather extensions.

I disabled all of the extensions other than uBO, as directed for troubleshooting, and the problem at BeforeItsNews.com still persisted.

### Screenshot(s)

<details>
<summary>Screenshot(s)</summary>

![uBO turned on - 'Now' counter does not appear](https://github.com/uBlockOrigin/uAssets/assets/165850681/cb7a1951-2b73-41a7-8edc-d9d9c723b206)

![uBO turned OFF - 'Now' counter DOES appear](https://github.com/uBlockOrigin/uAssets/assets/165850681/f3472112-9f48-4b48-a1ff-37e99ec1abf5)

</details>

### Configuration

<details>

```yaml
uBlock Origin: 1.56.0
Firefox: 115
filterset (summary):
 network: 128205
 cosmetic: 45751
 scriptlet: 18962
 html: 1730
listset (total-discarded, last-updated):
 default:
  user-filters: 0-0, never
  ublock-filters: 37466-138, 1h.54m Δ
  ublock-badware: 8131-15, 1h.54m Δ
  ublock-privacy: 777-0, 1h.54m Δ
  ublock-unbreak: 2297-19, 1h.54m Δ
  ublock-quick-fixes: 214-0, 1h.54m Δ
  easylist: 84558-763, 1h.54m Δ
  easyprivacy: 50425-4, 1h.54m Δ
  urlhaus-1: 8920-5, now
  plowe-0: 3764-809, now
filterset (user): [empty]
userSettings: [none]
hiddenSettings: [none]
supportStats:
 allReadyAfter: 1380 ms (selfie)
 maxAssetCacheWait: 494 ms
 cacheBackend: indexedDB
popupPanel:
 blocked: 55
 network:
  beforeitsnews.com: 6
  d2znr2yi078d75.cloudfront.net: 1
  doubleclick.net: 6
  google.com: 8
  googletagmanager.com: 1
  mgid.com: 12
  nerangee.com: 2
  sharethis.com: 1
  translate.googleapis.com: 2
  youtube.com: 16
 extended:
  ##[src*="/banner"]
  ##.rotating_text_link
  ##[href*="hop.clickbank.net"]
  ##[href^="https://tinyurl.com/"]
  ##[id^="banners_"]
  ##div[id*="ScriptRoot"]
  ##+js(trusted-replace-node-text, script, /^(\(function serverCon…
  ##+js(set-constant, ytInitialPlayerResponse.playerConfig.ssapCon…
  ##+js(set-constant, ytInitialPlayerResponse.streamingData.server…
  ##+js(json-prune, playerResponse.playerConfig.ssapConfig playerR…
  ##+js(set-constant, ytInitialPlayerResponse.playerAds, undefined…
  ##+js(set-constant, ytInitialPlayerResponse.adPlacements, undefi…
  ##+js(set-constant, ytInitialPlayerResponse.adSlots, undefined)
  ##+js(set-constant, playerResponse.adPlacements, undefined)
  ##+js(set-constant, rwt, noopFunc)
```
</details>

**Comment by stephenhawk8054** [comment]
It's the website tracker `https://ajax.beforeitsnews.com/core/ajax/counter/count.php`

You can add this as your custom filter

```adb
@@||ajax.beforeitsnews.com/core/ajax/counter/count.php$xhr,1p
```

**Comment by FewThereBe** [comment]
That worked! Thank you! God bless!

**Comment by stephenhawk8054** [comment]
Hmm... The site sends user's UUID in the request data in the XHR connection. I'm not sure if it should be unblocked.

**Comment by FewThereBe** [comment]
I'm not exactly sure what that means (or the ramifications; I'm no expert at this, just an end-uesr), but I need to see that real-time counter number. It is an important metric for monitoring story interest in real-time.
